tests/recipient_validation/test_phone_number.py

```diff
@@ -564,6 +572,13 @@ def test_international_does_not_normalise_to_uk_number(self, phone_number, expec
             PhoneNumber(phone_number, allow_international=False)
         assert exc.value.code == expected_error_code
 
+    @pytest.mark.parametrize("phone_number", valid_channel_island_numbers)
```

Comment:
Not sure if we need an explicit test here. To my mind we shouldn't be indirectly testing library code. Obviously this gives us peace of mind right now that the bug is fixed for some of the patterns listed as an issue, but we aren't actually testing our code here, rather a dependency.

Reply:
That is true - I'm happy to not have this unit test 🙌🏼

Reply:
If the library broke this again, would we want to know about it before we upgraded the dependency to the newest version? I would argue yes.

Reply:
I guess the problem is ... there are lots of ways this can break. It didn't break for all Jersey telephone numbers, and it's not really possible to predict what _types_ of numbers it will be broken for in the future.